src/test/java/com/conveyal/gtfs/graphql/GTFSGraphQLTest.java

```diff
@@ -75,204 +77,250 @@ public static void setUpClass() throws IOException {
         validate(testInjectionNamespace, testInjectionDataSource);
     }
 
-    @AfterClass
+    @AfterAll
     public static void tearDownClass() {
         TestUtils.dropDB(testDBName);
         TestUtils.dropDB(testInjectionDBName);
     }
 
     /** Tests that the graphQL schema can initialize. */
-    @Test(timeout=5000)
```

Comment:
Any reason not to use the `@Timeout` annotation here? I'm not actually sure what this timeout does or if it's needed. Did you try just removing the timeout code entirely (and did it have any negative effects)?

Reply:
@landonreed I have since tested minus the timeout assertion and all is fine. So have permanently removed.

Reply:
The timeouts in here serve a purpose which is to make sure that the GraphQL queries don't take forever to execute. It is sort of a performance test.